tests/authorisation-adjustment/admin-panel.spec.js

```diff
@@ -0,0 +1,9 @@
+// @ts-check
```

Comment:
Suggestion:

Can we have comments here as well so its clear what case is tested. e.g // Verify page title

Reply:
This test is not doing much except that it verifies that the admin panel is there. To-be-fair we can leave it out too
We'll add the E2E logic once we pick-up https://github.com/adyen-examples/adyen-testing-suite/issues/24